test/evergreen.yml

```diff
@@ -4111,6 +4111,71 @@ buildvariants:
   tasks:
     - name: ".pull_request_compilers"
 
+- name: ubuntu2004-not-standalone
+  display_name: "Ubuntu 20.04 Not Standalone"
+  run_on:
+  - ubuntu2004-test
+  expansions:
+    posix_configure_flags: -DWT_STANDALONE_BUILD=0
+    test_env_vars:
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_TOPDIR/TCMALLOC_LIB/lib
+    CMAKE_PREFIX_PATH: -DCMAKE_PREFIX_PATH="$(pwd)/TCMALLOC_LIB"
+    CMAKE_INSTALL_PREFIX: -DCMAKE_INSTALL_PREFIX=$(pwd)/cmake_build/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
+    cmake_generator: Ninja
+    make_command: ninja
+    data_validation_stress_test_args: -t r -m -W 3 -D -p -n 100000 -k 100000 -C cache_size=100MB
+  tasks:
+    - name: compile
+    - name: make-check-test
+    - name: unit-test
+    - name: configure-combinations
+    - name: syscall-linux
+    - name: checkpoint-filetypes-test
+    - name: unit-test-zstd
+    - name: unit-test-random-seed
+    - name: spinlock-gcc-test
+    - name: spinlock-pthread-adaptive-test
+    - name: compile-wtperf
+    - name: ftruncate-test
+    - name: long-test
+    - name: unit-test-long
+      distros: ubuntu2004-large
+    - name: static-wt-build-test
+    - name: linux-directio
+      distros: ubuntu2004-build
+    - name: format-mirror-test
+    - name: format-smoke-test
+    - name: format-failure-configs-test
+    - name: data-validation-stress-test-checkpoint
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s1
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s1-no-timestamp
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s2
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s3
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s3-no-timestamp
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s4
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s5
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s5-no-timestamp
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s6
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s7
+    - name: data-validation-stress-test-checkpoint-no-timestamp
+    - name: unittest-test
+    - name: tiered-storage-extensions-test
+    - name: ".stress-test-1"
```

Comment:
Note I have duplicated all the stress test jobs from the stress test variant here.